exdb/tests/integration_tests.py

```diff
@@ -345,8 +350,8 @@ def test_week_ahead(self):
                                          audience="b",
                                          status="ad",
                                          attendance=3)
-        response = self.clients['ra'].get(reverse('ra_home'))
-        self.assertEqual(len(response.context["week_ahead"]), 1, "There should be 1 experience in the next week")
+        response = self.clients['ra'].get(reverse('home'))
+        self.assertEqual(len(response.context["upcoming"]), 1, "There should be 1 experience in the next month")
```

Comment:
We should override `settings.HALLSTAFF_TIME_AHEAD` and `settings.RA_TIME_AHEAD` in order to keep these tests consistent regardless of what people have in their `settings.py`.